packages/vcvrack/PKGBUILD

```diff
@@ -88,31 +83,31 @@ prepare() {
     --pkgdesc "$pkgdesc" \
     --genericname "Virtual modular synthesizer" \
     --categories "AudioVideo;Audio"
-
-  cd ../$_plugin_name-$_plugin_ver
-  # fix build error about a class that was moved to the SDK
-  patch -p1 -i ../917628e.patch || true
 }
 
 build() {
-  cd $_name-$pkgver
-  _ldflags="-shared -ldl \
+  cd $_name
+  _ldflags="-Wl,--whole-archive \
+    dep/lib/librtaudio.a \
+    -Wl,--no-whole-archive \
+    -shared -ldl \
     $(pkg-config --libs glew \
     glfw3 jansson libcurl openssl \
     libarchive libzstd speexdsp \
-    samplerate rtmidi rtaudio)"
+    samplerate rtmidi \
+    alsa jack libpulse libpulse-simple)"
   VERSION=$pkgver make -C dep includes
   VERSION=$pkgver make LDFLAGS+="$_ldflags" STANDALONE_LDFLAGS="$LDFLAGS"
   cd ../$_plugin_name-$_plugin_ver
-  VERSION=$_plugin_ver RACK_DIR=../$_name-$pkgver \
+  VERSION=$_plugin_ver RACK_DIR=../$_name \
     LDFLAGS+=" $(pkg-config --libs samplerate)" make dist
 }
 
 package() {
   # Rack does not start with glfw-wayland
-  depends+=(libcurl.so libGLEW.so libarchive.so libcrypto.so librtaudio.so librtmidi.so
+  depends+=(libcurl.so libGLEW.so libarchive.so libcrypto.so librtmidi.so
```

Comment:
ah, I forgot to add transitive dependencies from rtaudio, i.e. alsa/jack/pulse. Wondering though if all these are hard requirements for Rack to work (as suggested by the rtaudio package) or if they're actually optional, so it works with either of the 3.